Products/ZenUI3/browser/resources/js/zenoss/ExtOverrides.js

```diff
@@ -93,6 +93,10 @@ Ext.override(Ext.util.Sorter, {
         border: false
     });
 
+    Ext.override(Ext.menu.Menu, {
```

Comment:
Why has this been added?

Reply:
The reason is that regarding the style guide attached in ZEN-29844 our drop-down menu should have borders. 
The `Ext.menu.Menu` inherits `Ext.panel.Panel` and on 92 line of file `ExtOverrides.js` we have an expression that disabled borders for `Ext.panel.Panel` in accordance `Ext.menu.Menu` won't have borders as well.
I did not want to create a chain when we disable borders via javascript and then enable it via CSS `!important` so I decided to do that in this way.